lib/driver/runtimeTool.cpp

```diff
@@ -85,6 +85,8 @@ ToolOnModule(WasmEdge::VM::VM &VM, const std::string &FuncName,
   if (auto Result = AsyncResult.get()) {
     /// Print results.
     for (size_t I = 0; I < Result->size(); ++I) {
+      auto HTCode = (*Result)[I].second.getHeapTypeCode();
+
```

Comment:
You should not declare this variable here. It will create a temporary variable; however, most cases won't use it.